community-content/pytorch_text_classification_using_vertex_sdk_and_gcloud/pytorch-text-classification-vertex-ai-train-tune-deploy.ipynb

```diff
@@ -1834,7 +1834,7 @@
         "%%writefile ./custom_container/Dockerfile\n",
```

Comment:
<p>(Not required for approval) Any thoughts on beginning to use Artifact Registry?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='257'/>

Reply:
<p>I think we should begin using Artifact Registry because <a href="https://cloud.google.com/artifact-registry/docs/transition/transition-from-gcr" target="_blank">Container Registry will only receive critical security fixes</a></p><p> </p><p>I will submit another PR with steps to GAR.</p>

Reply:
Yup, we should be using GAR moving forward